scapy/volatile.py

```diff
@@ -250,9 +250,14 @@ def _fix(self):
     def __mul__(self, n):
         return self._fix()*n
 
-class RandBin(RandString):
+class RandBin(RandString, object):
```

Comment:
I think a "string" in Scapy's spirit (if such a thing exists) is actually a `bytes` object in Python 3 (see `Str*Field`s). So maybe `RandString._fix()` should return a `bytes` object. What do you think?

Reply:
Not sure about this, but i'm more to keep current behavior because:
- many other volatile values that are returning str (definatly not bytes), such as RandIP, RandMAC, are subclasses of RandString. It would make no sense if those returned `str` and RandStr `bytes`
- RandBin already exists

Reply:
A string, in Python 3, means a unicode string. A RandString must return bytes.

Reply:
Got it. Should `RandIP` still inherit from `RandString` ?